src/main/webapp/app/core/course/manage/user-management-dropdown/user-management-dropdown.component.spec.ts

```
@@ -0,0 +1,73 @@
+                { provide: RouterLink, useClass: MockRouterLinkDirective },
+                { provide: TranslateService, useClass: MockTranslateService },
+                { provide: ActivatedRoute, useValue: new MockActivatedRoute() },
+            ],
+        }).compileComponents();
+    });
+
+    beforeEach(() => {
+        fixture = TestBed.createComponent(UserManagementDropdownComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)